src/core/Text/Text.md

```diff
@@ -2,11 +2,16 @@
 import { Paragraph, Text } from 'suomifi-ui-components';
 
 <>
+  <Paragraph>Paragraph text</Paragraph>
   <Paragraph marginBottomSpacing="m">
     <Text.lead>Leading text</Text.lead>
   </Paragraph>
   <Paragraph>
     <Text>Body text</Text> <Text.bold>Bold text</Text.bold>
   </Paragraph>
+  <Paragraph>
```

Comment:
Are these changes related to this PR or is this an additional fix/improvement?

Reply:
Documentation is found from the commit message https://github.com/vrk-kpa/suomifi-ui-components/pull/211/commits/e4d037458892e2f37baa0708ead6c8517237d751

If the change cannot be included here make a request change for this PR.